pyproject.toml

```diff
@@ -86,3 +93,5 @@ package = "pipx"
 [[tool.mypy.overrides]]
 module = ["pycowsay.*"]
 ignore_missing_imports = true
+
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
```

Comment:
I guess I better just remove that comment since file could be auto resorted and comments disappear

```suggestion
```